hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -464,4 +465,22 @@ public static ProcessingGuarantee min(ProcessingGuarantee g1, ProcessingGuarante
     public static <T, R> List<R> toList(@Nonnull Collection<T> coll, Function<? super T, ? extends R> mapFn) {
         return coll.stream().map(mapFn).collect(Collectors.toList());
     }
+
+    /**
+     * Extracts the file name from the URL. File name is the part of {@code
+     * url.getPath()} after the last '/' character. Returns empty string if the
+     * path ends with a '/'.
+     * <p>
+     * Returns null if input is null or if {@code url.getPath()} returns null.
+     */
+    @Nullable
+    public static String fileNameFromUrl(@Nullable URL url) {
```

Comment:
minor but I think better to put this into `IOUtil`?

Reply:
done